scripts/ethcommands.ts

```diff
@@ -117,6 +149,40 @@ export const bridgeToL3Command = {
   },
 };
 
+export const bridgeNativeTokenToL3Command = {
+  command: "bridge-native-token-to-l3",
+  describe: "bridge native token from l2 to l3",
+  builder: {
+    amount: {
+      string: true,
+      describe: "amount to transfer",
+      default: "10",
+    },
+    from: {
+      string: true,
+      describe: "account (see general help)",
+      default: "funnel",
```

Comment:
in my tests funnel doesn't have l3tokens initially, so it shouldn't be the default
set the default to bridge-native-token-to-l3.